tasks/config.yml

```diff
@@ -1,15 +1,15 @@
 ---
 
 - name: Check if modules are enabled
-  shell: "{{ openseadragon_drush_path }} pm-list --format=json"
+  shell: "{{ openseadragon_drush_path }} --root {{ openseadragon_composer_root }}/web pm-list --format=json"
   args:
     chdir: "{{ openseadragon_composer_root }}/web"
```

Comment:
These chdir lines become redundant when we add the root option; besides, they don't seem to work anymore anyway. Best to remove them to avoid confusion.